ccmlib/dse_node.py

```diff
@@ -67,7 +67,7 @@ def start(self,
               replace_token=None,
               replace_address=None,
               jvm_args=None,
-              wait_for_binary_proto=False,
+              wait_for_binary_proto=True,
```

Comment:
Worried about this API change; is it necessary? I don't know anything about pcmanus/ccm's DSE userbase, fwiw.